tests/helper/helper.go

```diff
@@ -131,6 +131,28 @@ func DeleteNamespace(t *testing.T, kc *kubernetes.Clientset, nsName string) {
 	assert.NoErrorf(t, err, "cannot delete kubernetes namespace - %s", err)
 }
 
+// Waits for a namespace to be deleted.
+func WaitForNamespaceToBeDeleted(t *testing.T, kc *kubernetes.Clientset, namespace string,
```

Comment:
I think that this could be included into `CreateNamespace()` so it runs by default for all tests.
@v-shenoy @JorTurFer WDYT?